macros/DOMxRef.ejs

```diff
@@ -21,13 +21,14 @@ let str = $1 || $0;
 let rtlLocales = ['ar', 'he', 'fa'];
 let localStrings = string.deserialize(await template("L10n:Common"));
 
-api = api.replace(' ', '_').replace('()', '').replace('.prototype.', '.').replace(/\./g, '/');
+api = api.replace(/ /g, '_').replace(/\(\)$|\(\)\./g, '').replace(/\.prototype\./g, '.').replace(/\./g, '/');
```

Comment:
Please simplify the parens case to replace all '()' pairs, regardless of position in the string. Or convince me that it is a bug. Note however, that replacing '().' with '' is wrong: we'd want the dot to remain in that case.

I'd be fine just doing regular non-global string replacement for () and .prototype. as in the old code, since I can't imagine a case where they would appear twice. That would make your fix even more minimal and lower risk. But I'll accept the regex version, too.